yast/src/modules/SMTData.rb

```diff
@@ -56,7 +56,7 @@ def main
 
       @smt_enabled = nil
 
-      @smt_services = ["smt.target"]
+      @smt_services = ["mysql", "smt", "apache2", "smt-schema-upgrade"]
```

Comment:
One thing why not add `mysql`, `apache2` and also `cron` to the smt service file as `Requires` this makes sure that smt is only running if **all** services where started successfully? What do you think?

Reply:
Oh wait the service file actually already requires `mysql` and starts the accordingly. Maybe `cron` and `apache2` can be also hooked up into there.

Reply:
The `smt.service` is not the actual service, it should have better been named "smt-maintenance.service". If you want to start SMT currently, you literally have to start apache2.service and mysql.service, there is no real "smt.service"

Reply:
@simson-suse maybe make `smt.service` a real service, then?

Reply:
@ikapelyukhin We could, but that not part of this bugfix, IMO. I wouldn't mind adding a feature card to create a service for SMT. Wouldn't we have to worry about backwards compatibility though? Because now, if anybody worked with SMT, they would expect `smt.service` to be a maintenance thing, and starting/stopping it wouldn't affect the SMT instance.

Reply:
@simson-suse Dunno, I'm actually not sure what the current service does. :grimacing:

Reply:
How about a `smt-server` service? I tought the common way to handle services dependencies should be in a service for the __actual__ service running.

Reply:
@felixsch Well, there was no `smt-server` for 10 years, so maybe it's not all that needed after all :D I just restart apache whenever I do stuff with SMT and that seems to work fine.

Reply:
Okay since @simson-suse also pointed out its not part of this bug fix. Let's leave it like this